bindings/java/Cargo.toml

```diff
@@ -82,6 +82,7 @@ services-all = [
   "services-wasabi",
   "services-mongodb",
   "services-sqlite",
+  "services-azfile",
```

Comment:
Please don't add service in default list.

Reply:
It is under `services-all` not `default`

Reply:
Sorry for my mistake🥵